benchexec/benchexec.py

```diff
@@ -152,6 +152,13 @@ def create_argument_parser(self):
                           metavar="N",
                           help="Limit each run of the tool to N CPU cores (-1 to disable).")
 
+        parser.add_argument("-s", "--allowedCores", dest="coreset",
```

Comment:
I would prefer going without a short option because this option is not one that will be used by many people (I guess).